tests/unit/oidc/test_views.py

```diff
@@ -383,18 +385,21 @@ def test_mint_token_from_oidc_no_pending_publisher_ok(
     time = pretend.stub(time=pretend.call_recorder(lambda: 0))
     monkeypatch.setattr(views, "time", time)
 
-    project = pretend.stub(
-        id="fakeprojectid",
-        record_event=pretend.call_recorder(lambda **kw: None),
+    project = Project(id="fakeprojectid")
+    monkeypatch.setattr(
+        project, "record_event", pretend.call_recorder(lambda **kw: None)
     )
-    publisher = pretend.stub(
-        id="fakepublisherid",
-        projects=[project],
-        publisher_name="fakepublishername",
-        publisher_url=lambda x=None: "https://fake/url",
+
+    publisher = github.GitHubPublisher(
```

Comment:
I had to do this change because I had to add [this change](https://github.com/pypi/warehouse/pull/14063/files#diff-00352684c58d85c29397c3a8b7a868b5a7846e75a3dfb6f67af1ab141483ff0bR204) to stop linting errors from occurring [here](https://github.com/pypi/warehouse/pull/14063/files#diff-00352684c58d85c29397c3a8b7a868b5a7846e75a3dfb6f67af1ab141483ff0bR236) and anywhere else a prop was accessed on the OIDCPublisher object.